tutorials/vbpr_text.ipynb

```diff
@@ -0,0 +1,240 @@
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "In this tutorial, we will employ a recommendation algorithm designed to leverage item images to work with item text content.\n",
+    "The model of interest is [VBPR](https://arxiv.org/abs/1510.01784) that makes use of visual features extracted from item images using pre-trained CNN deep networks.\n",
+    "\n",
+    "What if our data doesn't come with item images but text, for example, we are working with [MovieLens dataset](https://grouplens.org/datasets/movielens/) that contains text plots of the movies. At the same time, we also want to work with VBPR model which is not originally designed for text data. To address that challenge, Cornac provides mechanisms to cross between modalities, in which here we focus on crossing from text to image to utilize image models for text-available datasets.\n",
+    "\n",
+    "To quickly give an overview of the process, we are going to go through following steps for modality crossing:\n",
+    "1. Use `TextModality` to build representations (e.g., bag-of-words, TF-IDF) from raw text.\n",
+    "2. Construct `ImageModality`, which will be used by `VBPR` as well as other models working with images, from textual representations.\n",
+    "3. Split data using provided `RatioSplit` evaluation method.\n",
+    "4. Define models and metrics used to evaluate models' performance.\n",
+    "5. Test the models by running an experiment supported by Cornac\n",
+    "\n",
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install Cornac\n",
+    "!pip3 install numpy cornac\n",
+    "# install PyTorch for VBPR model\n",
+    "!pip3 install torch\n",
+    "\n",
+    "import cornac\n",
+    "from cornac.data import Reader\n",
+    "from cornac.datasets import movielens\n",
+    "from cornac.eval_methods import RatioSplit\n",
+    "from cornac.data import TextModality, ImageModality\n",
+    "from cornac.data.text import BaseTokenizer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare data\n",
+    "Here we use MovieLens 100K dataset as already provided inside Cornac library. We load movie plots with their corresponding ids, then load the rating data filtered the movies that don't come with plots."
```

Comment:
Here we use `the` MovieLens 100K dataset, which is already accessible from Cornac. Hence, we can simply load the movie plots and user-movie rating data as follows: